packages/cozy-notifications/src/notifications.spec.js

```diff
@@ -68,7 +68,7 @@ describe('notifications', () => {
     return { client, notificationView }
   }
 
-  it('should send a notification view', async () => {
+  xit('should send a notification view', async () => {
```

Comment:
to be removed?

Reply:
As I mention in the PR description, I disable 5 tests in cozy-notifications. Problem is the tests didn't mock correctly to prevent failing for external reasons. We need to investigate this (I will create a Github issue)

Reply:
If we don't fix it now, we'll never fix it later. 

Please share the stacktrace of the fail to see how we can fix it.

Reply:
fixed